cmd/kpod/create.go

```diff
@@ -264,10 +264,11 @@ func parseCreateOpts(c *cli.Context, runtime *libpod.Runtime) (*createConfig, er
 		return &createConfig{}, errors.Wrapf(err, "unable to process labels")
 	}
 	// ENVIRONMENT VARIABLES
-	env, err := getAllEnvironmentVariables(c.StringSlice("env-file"), c.StringSlice("env"))
+	env, err := readKVStrings(c.StringSlice("env-file"), c.StringSlice("env"))
 	if err != nil {
 		return &createConfig{}, errors.Wrapf(err, "unable to process environment variables")
 	}
+	env = append(defaultEnvVariables, env...)
```

Comment:
Nit: this seems a little backward - usually would do append(env, defaultEnvVariables...) to make it more clear. Doesn't actually effect the results just a style preference

Reply:
I did it this way so that if path is defined in defaultEnvVariables and in --env path, then the second path will replace the first path.

If I reverse then the user would not be able to override the defaults.